frontend/src/scss/PrimeStyles.scss

```diff
@@ -67,14 +67,44 @@
 
 .prime-queue-item {
   padding: 0;
-  // margin-bottom: 1rem;
   position: relative;
+  .result-ready {
```

Comment:
There's a `QueueItem.scss` where this could technically all be put, since it's specific to the queue. The complication is that our SCSS files are currently not factored in a way that allows us to get the variable names like `$theme-color-warning`. So it needs to stay here for now I suppose. My one suggestion for now would be to change the class names to `sr-` instead of `prime-`.

Reply:
I'm new to this, what's the difference between the two prefixes?

Reply:
>The complication is that our SCSS files are currently not factored in a way that allows us to get the variable names like $theme-color-warning. 

@dmethvin-gov Are you referring to the fact that we can't retrieve variable names in React component SCSS?

Reply:
@maya 

The project was originally named after the overarching PRIME but now its called SimpleReport so it would be nice to use that instead. Also some of the early classes are not too descriptive like `prime-li`. Not that some of the later stuff is _that_ much better (we never really settled on a common approach) but at least once styles are in their own component-based files it should be easier to refactor in small chunks without breaking everything or causing ugly rebase conflicts.

Right now we include all the variable setting and utility classes directly in App.scss along with some global styles. If we split the non-css-rule definitions into a definitions.scss or similar name we could have each one of the React component files just include that file at the top and get all the goodies that way.

Reply:
Thanks for the history. I wasn't aware of their purpose and have just been writing them prefix-less. What's the reason for using prefixes? Do we use any other libraries that might conflict with them or is it just for good measure?

Previously, I was looking for a way to do that React component SCSS but didn't see an existing approach related to pulling in the goodies I needed. Will be happy to update/refactor once something is set up. I'm a big user of the USWDS utility functions so would need a way to access those.

Reply:
Yeah I think it was done to avoid collisions. I'm guilty of cargo-culting on this one because it had started before I arrived and I just went with it rather than trying to achieve consensus with everyone. With per-component CSS it's usually enough to have a top-level component with a prefixed name and then let all the other internal classes be simple, since the `.archived` in `.sr-user-listing .archived` is unambiguous unless someone unwisely created a global `.archived`.

Refactoring the SCSS to make it consistent would be great, especially because it would give us access to the utilities. I'll see if i can get something started on that. It should be possible to fix App.scss just enough for us to do new stuff a better way without having to refactor everything immediately.

Reply:
I might punt on the renaming since, when you mentioned changing from `prime-` to `sr-` I was thinking it would _then_ be inconsistent with the other classes. I think we just need a refactor ticket.